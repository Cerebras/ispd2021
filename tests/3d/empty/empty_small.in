# Dimension
3
# Volume X, Y
4 4 1
# Fabric
4 4
# Cost params
1 0 
# Heatmap
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0