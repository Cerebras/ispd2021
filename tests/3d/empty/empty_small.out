# Inverse sampling step
10
# Prisms
-1 0 0 0 4 4 1
compute_map:
adapter_map:
